cassovary-core/src/main/scala/com/twitter/cassovary/graph/Traverser.scala

```diff
@@ -335,7 +335,7 @@ trait QueueBasedTraverser[+V <: Node] extends Traverser[V] {
       degreeLimit.isLimitReached(node.neighborCount(dir))) {
       Seq()
     } else {
-      limitAddedToQueue(node.neighborIds(dir).filter(n => shouldBeEnqueued(coloring.get(n))))
+      limitAddedToQueue(node.neighborIds(dir).toSeq.filter(n => shouldBeEnqueued(coloring.get(n))))
```

Comment:
1. filter: dunno which will be more expensive: (a) converting to Seq, or (b) counting the number of filtered items and then allocating an array for it and putting the items in.
2. (A) If `CSeq[T]` is efficiently implemented (Int or Long), will it be okay? If so, we are good?